src/main/java/mate/academy/springboot/datajpa/dto/mapper/ProductMapper.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+import mate.academy.springboot.datajpa.dto.product.ProductRequestDto;
+import mate.academy.springboot.datajpa.dto.product.ProductResponseDto;
+import mate.academy.springboot.datajpa.model.Product;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductMapper implements MapperDto<ProductRequestDto, ProductResponseDto, Product> {
+    private final CategoryService categoryService;
+
+    public ProductMapper(CategoryService categoryService) {
+        this.categoryService = categoryService;
+    }
+
+    @Override
+    public ProductResponseDto toDto(Product product) {
+        ProductResponseDto productResponseDto = new ProductResponseDto();
+        productResponseDto.setId(product.getId());
+        productResponseDto.setTitle(product.getTitle());
+        productResponseDto.setPrice(product.getPrice());
+        productResponseDto.setCategoryId(product.getCategory().getId());
```

Comment:
do you really need to return Category with product? Don't forget that Category is lazy field